corehq/apps/userreports/README.md

```diff
@@ -361,6 +362,29 @@ The from_date_expression and to_date_expression can be any valid expressions, or
 }
 ```
 
+#### "Evaluator" expression
+`evaluator` expression can be used to evaluate statements that contain arithmetic (and simple python like statements). It evaluates the statement specified by `equation_statement` which can contain variables as defined in `variables_expression`.
+
+```json
+{
+    "type": "evaluator",
+    "equation_statement": "a + b - c + 6",
+    "variables_expression": {
+        "type": "dict",
+        "properties": {
+            "a": 1,
+            "b": 20,
+            "c": 2
+        },
+    }
+}
+```
+This returns 25 (1 + 20 - 2 + 6).
+
+`equation_statement` can be any statement that returns a valid number. All python math [operators](https://en.wikibooks.org/wiki/Python_Programming/Basic_Math#Mathematical_Operators) except power opertor are available for use.
+
+`variables_expression` is a [Dict Expression](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/README.md#dict-expressions) where keys are names of variables used in the `equation_statement` and values are values of those variables. Variables can be any valid numbers (Python datatypes `int`, `float`, and `long` are considered valid numbers.) or also expressions that return numbers.
```

Comment:
I think it'd be easier to make the property a `dict` of strings to expressions, rather than a Dict Expression.